src/NzbDrone.Core/Datastore/Migration/146_naming_config_colon_replacement_format.cs

```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(146)]
+    public class naming_config_colon_action : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("NamingConfig").AddColumn("ColonReplacementFormat").AsString().WithDefaultValue("");
+            Update.Table("NamingConfig").Set(new { ColonReplacementFormat = "" }).AllRows();
```

Comment:
Is this line needed, won’t they all default to “” when you add the column with line 11